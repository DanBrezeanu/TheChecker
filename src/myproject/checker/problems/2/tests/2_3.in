106520667 895639915 87164104 415157861 713313280 217367725 182868245 128301864 534549782 488562 997664750 349994387 627796912 339357058 942910213 508848731 258784408 871594952 760103962 842459205 692545256 74717063 34459837 150796722 80948195 590650079 671010229 779586835 396109159 479694879 752817029 650859441 599237315 722113583 935302598 390377438 120637977 787628076 926942985 504967355 881339207 497205858 785726060 283906110 705585795 241129052 92101952 0
