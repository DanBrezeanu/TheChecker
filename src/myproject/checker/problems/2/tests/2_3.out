484563750.29
